### PR TITLE
APIS-5835 - Phase 11 changes

### PR DESCRIPTION
**Pull request by peteslater-ee**

### CODE REVIEW

File: app/uk/gov/hmrc/gatekeeper/views/deploymentApproval/DeploymentApprovalView.scala.html [hunk 1]
```diff
@@ -20,66 +20,60 @@
...
-            <div class="grid-layout__column--1-3 text--center">
-                @if(unapprovedApis.isEmpty) {"There are no unapproved APIs."}
-            </div>
-        </div>
-
-    </form>
-
-    <div class="form-group>">
-        <a class="back-to-top-link" href="#">Back to Top</a>
+    <div class="govuk-visually-hidden" style="margin-left:0">
```
Comment (johnsgp) [inline app/uk/gov/hmrc/gatekeeper/views/deploymentApproval/DeploymentApprovalView.scala.html:71]: Was the `js-hidden` class linked to some JavaScript?

File: app/uk/gov/hmrc/gatekeeper/views/applications/ApplicationView.scala.html [hunk 2]
```diff
@@ -505,8 +505,8 @@ <h1 class="govuk-heading-l">@application.name</h1>
         @if(application.collaborators.nonEmpty) {
           <span id="all-team" class="govuk-visually-hidden" aria-hidden="true">@EmailsFormatter.format(application.collaborators)</span>
           <span id="all-admin" class="govuk-visually-hidden" aria-hidden="true">@EmailsFormatter.format(application.collaborators, Some(CollaboratorRole.ADMINISTRATOR))</span>
-          <button class="govuk-button govuk-!-margin-bottom-3" onclick="copyText('all-team')" data-clip-text>Copy all team member email addresses</button>
-          <button class="govuk-button govuk-!-margin-bottom-0" onclick="copyText('all-admin')" data-clip-text>Copy admin email addresses</button> 
+          <button id="copy-users-to-clip" class="govuk-button govuk-!-margin-bottom-3" onclick="copyText('all-team')" data-clip-text>Copy all team member email addresses</button>
+          <button id="copy-users-to-clip" class="govuk-button govuk-!-margin-bottom-0" onclick="copyText('all-admin')" data-clip-text>Copy admin email addresses</button>
```
[the remark this answers is not in the document]
Comment (peteslater-ee) [reply app/uk/gov/hmrc/gatekeeper/views/applications/ApplicationView.scala.html:509]: I think it should be but it wasn't in Ryans original work - I'll change them.

File: app/uk/gov/hmrc/gatekeeper/views/deploymentApproval/DeploymentApprovalView.scala.html [hunk 3]
```diff
@@ -20,66 +20,60 @@
...
+        </tr>
+      </thead>
+      <tbody  class="govuk-table__body">
+        @for((api, index) <- unapprovedApis.zipWithIndex) {
+          <tr role="row" class="govuk-table__row">
+            <td class="govuk-table__cell" id="api-servicename-@index">@api.serviceName</td>
+            <td class="govuk-table__cell" id="api-name-@index">@api.name</td>
+            <td class="govuk-table__cell" id="api-environment-@index">@api.env</td>
+            <td class="govuk-table__cell">
+              <a data-review- @api.serviceName class="govuk-link" href="@{routes.DeploymentApprovalController.reviewPage(api.serviceName, api.environment.get.toString)}">
```
[the remark this answers is not in the document]
Comment (peteslater-ee) [reply app/uk/gov/hmrc/gatekeeper/views/deploymentApproval/DeploymentApprovalView.scala.html:62]: Well spotted - I'll fix it.
I think it was added after Ryans original branch so it's removed it for me :(

File: app/uk/gov/hmrc/gatekeeper/views/deploymentApproval/DeploymentApprovalView.scala.html (hunk 1 above)
Comment (peteslater-ee) [reply app/uk/gov/hmrc/gatekeeper/views/deploymentApproval/DeploymentApprovalView.scala.html:71]: Quite possibly.

**Review by johnsgp — APPROVED** [review]

**Pull request merged**